readme.md

```diff
@@ -68,10 +68,11 @@ If you think we missed a plugin reference, please file an issue on [Github](http
 * Semantic Linkbacks (Required) - makes IndieWeb comments and mentions look better on your site.
 * Post Kinds - Allows you to reply/like/RSVP etc to another site from your own, by adding support for kinds of posts to WordPress.
 * Syndication Links - Adds fields to a post to allow manual entry of syndication links as well as automatically from a supported syndication plugin. Fully supports Social, partial support for NextScripts: Social Networks Auto-Poster (aka SNAP).
-* MicroPub - A MicroPub Server
+* MicroPub - A MicroPub Server for publishing posts using Micropub clients
```

Comment:
would the inclusion of "third party" Micropub clients help people new to this understand that it requires another tool to publish?